CUSW-chula/cusw-frontend
# feat/boom-task-workspace

**Changes Made**
- Added a button to trigger file upload.
- Implemented a hidden input field to handle file selection.
- Created a callback function to process the uploaded file.

**How to Test**
1. Navigate to the component in the application.
2. Click the "Upload" button.
3. Select a file from your device.
4. Confirm that the file upload is processed correctly and logged in the console.
